ballerina/io_error.bal

```diff
@@ -14,16 +14,20 @@
 // specific language governing permissions and limitations
 // under the License.
 
+import ballerina/jballerina.java;
+
 # Represents IO module related errors.
 public type Error distinct error;
 
 # This will return when connection timed out happen when try to connect to a remote host.
 public type ConnectionTimedOutError distinct Error;
```

Comment:
Don't we need to mark this error as well? Not sure whether we are returning such error

Reply:
No this error is not used in the Java side.

AFAIU we are not using this error at all (could not find any code-references), are we safe to remove this error @daneshk ?

Reply:
Looks like this is not used. We will review and remove it.